src/site/content/en/blog/building-a-settings-component/index.md

```diff
@@ -964,4 +964,7 @@ to the [Community remixes](#community-remixes) section below!
 
 ## Community remixes
 
-<i>awaiting submissions!</i>
+- [@tomayac](https://twitter.com/tomayac) with a taste expression regardings the
```

Comment:
Can you rephrase this? I don't really get what "a taste expression" means, and there's a typo in "regarding**s**"